docs/web/index.md

```diff
@@ -585,10 +587,11 @@ or even another instance of the service.
 Most likely, this will happen when a business entity is stored based on a request and loaded later
 for further processing by a scheduler or due to a new user interaction.
 In this case, the `DetachedMetadataContext` must be persisted along with the entity and recreated
-when the entity is loaded.
-The `DetachedMetadataContext` can be defined as field in any MongoDB entity.
-
-Services that handle requests or messages in parallel must transfer the metadata context to their
-`Runnable` or `Callable` with `MetadataContext.transferMetadataContext(…)`.
-In most cases, developers should prefer`ContainerRequestContextHolder.transferRequestContext(…)`,
-which also transfers the metadata context.
\ No newline at end of file
+when the entity is loaded. The `DetachedMetadataContext` can be defined as field in any MongoDB
+entity.
+So, for services that handle requests or messages in parallel, the
+class [ContextCopyTaskDecorator](../../sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/async/ContextCopyTaskDecorator.java)
+will copy the
+metadata context attributes to the new threads, if
+the `org.springframework.boot.autoconfigure.task.TaskExecutionAutoConfiguration` class is
```

Comment:
Do we need to mention the details and the links to the classes? As we value auto configuration and bring the neccessary dependencies, a service developer has to do nothing but add `@Async` to the method. So I would rather document that the metadata context is automatically transferred to new threads for multithreaded processing, if `@Async` is used. That is similar to [the docs about async above](https://github.com/SDA-SE/sda-spring-boot-commons/blob/a4c10a9ca4c5b86691d9485ddb5d7585732429d2/docs/web/index.md#async) - so could be even shorter.